routes/index.js

```diff
@@ -1,9 +1,58 @@
 var express = require('express');
+const { response } = require('../app');
```

Comment:
You cant do this. This introduces a circular dependency which will BLOW SHIT UP ;)

Reply:
Everything else looks good tho

Reply:
Sounds good, I got rid of all the unused global variables in index.js should be good now.